src/main/java/study/racinggame/domain/Cars.java

```diff
@@ -28,4 +32,25 @@ public static Cars copyCars(Cars cars) {
   private static Car copyCar(Car car) {
     return new Car(car.name(), car.distance());
   }
+
+  public void forwardCars() {
+    cars.forEach(car -> car.forward(new RacingGameStrategy()));
+  }
```

Comment:
```suggestion
  public void forwardCars(GameStrategy strategy) {
    cars.forEach(car -> car.forward(strategy));
  }
```
- Cars 객체도 전략을 외부에서 주입받는게 자연스럽지 않나 생각합니다. RacingGame의 전략을 RacingGame에서 다루는 것이 적절치 않나..

```java
public class RacingGame {
  private final GameStrategy strategy;
  private final Cars cars;

  public RacingGame(List<String> carNames) {
    this(carNames, new RacingGameStrategy());
  }
  
  public RacingGame(List<String> carNames, GameStrategy strategy) {
    this.cars = Cars.newInstance(carNames);
    this.strategy = strategy;
  }
...
```